app/models/account_suggestions/similar_profiles_source.rb

```diff
@@ -51,7 +51,8 @@ def get(account, limit: DEFAULT_LIMIT)
     recently_followed_account_ids = account.active_relationships.recent.limit(5).pluck(:target_account_id)
 
     if Chewy.enabled? && !recently_followed_account_ids.empty?
-      QueryBuilder.new(recently_followed_account_ids, account).build.limit(limit).hits.pluck('_id').map(&:to_i).zip([key].cycle)
+      ids_from_es = QueryBuilder.new(recently_followed_account_ids, account).build.limit(limit).hits.pluck('_id').map(&:to_i)
```

Comment:
I think we don't have a spec for this similar profiles source -- might be good to add, although a good chunk of it might just be stubbing out Chewy/QueryBuilder..